draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1114,7 +1114,31 @@ The work on this draft was started at OAuth Security Workshop 2022 in Trondheim,
 
 # IANA Considerations {#iana_considerations}
 
-TBD
+## JSON Web Token Claims Registration
+
+This specification requests registration of the following Claims in the
+IANA "JSON Web Token Claims" registry [@IANA.JWT] established by [@!RFC7519].
+
+Object Property Disclosure Digests:
```

Comment:
do we need these headers `Object Property Disclosure Digests:`, `Array Element Disclosure Digest:`, `Disclosure Digest Hash Function:`..? they are the same as `claim description` values rn.

Reply:
They are there to breakup the sets of bullets for each claim that run together without something. And I honestly don't know how else to do it with the markdown to xml to html/txt/etc tool chain in place. 

So we kinda need something but it doesn't have to be these exactly.

Reply:
I just saw this example https://www.rfc-editor.org/rfc/rfc7515#section-9.1.2, but yeah.
maybe use claim names as the `breakups`?

Reply:
actually, maybe `<br/>` can be used to breakup with no text needed ....

Reply:
that seems to work ok